quinn-proto/src/connection.rs

```diff
@@ -3186,6 +3210,12 @@ struct SentPacket {
 /// Ensures we can always fit all our ACKs in a single minimum-MTU packet with room to spare
 const MAX_ACK_BLOCKS: usize = 64;
 
+/// Events to be sent to the Endpoint
+pub enum EndpointEvent {
```

Comment:
Eventually I'd like this to be wrapped inside an opaque struct wherever it occurs in a public API; unlike `Io` and `Event`, external code should never need to see into this enum.